cylc/flow/workflow_files.py

```diff
@@ -556,26 +559,38 @@ def register(
     # flow.cylc must exist so we can detect accidentally reversed args.
     source = os.path.abspath(source)
     check_flow_file(source, symlink_suiterc=True, logger=None)
-    symlinks_created = make_localhost_symlinks(
-        get_workflow_run_dir(flow_name), flow_name)
-    if bool(symlinks_created):
-        for src, dst in symlinks_created.items():
-            LOG.info(f"Symlink created from {src} to {dst}")
+    if not is_installed(get_workflow_run_dir(flow_name)):
+        symlinks_created = make_localhost_symlinks(
+            get_workflow_run_dir(flow_name), flow_name)
+        if bool(symlinks_created):
```

Comment:
```suggestion
        if symlinks_created:
```
No reason to use explicit `bool()` here, is there?